vision/google/cloud/vision/_http.py

```diff
@@ -49,7 +50,7 @@ def annotate(self, image, features):
         api_response = self._connection.api_request(
             method='POST', path='/images:annotate', data=data)
         responses = api_response.get('responses')
-        return responses[0]
+        return Annotations.from_api_repr(responses[0])
```

Comment:
Ditto here on `len(responses) == 1`. (I realize that is unrelated to this actual change)